### ISSUE TITLE
Cover not found and can't download uploaded files

### ISSUE THREAD

**Issue by kplaczek** [janeczku/calibre-web#1110]
I installed calibre web on Raspberry Pi 2B running raspbrian stretch. It went ok, everything is running except I can not see any of the uploaded covers. They exist in the directory with the book file. There is an 404 in the browser with the cover request. The same is for the book files. All of then yields 404. Can I have a hint how to get this fixed?

**Comment by OzzieIsaacs** [comment]
I just checked it on a new installation of calibre-web master on a raspberry pi zero. I only could check it with raspian Buster:
![grafik](https://user-images.githubusercontent.com/6438895/70931471-d65e0f00-2037-11ea-90e0-774cd2870c32.png)

The folder structure is as follows:
![grafik](https://user-images.githubusercontent.com/6438895/70931058-e2959c80-2036-11ea-8ddb-343fa2317bfd.png)
![grafik](https://user-images.githubusercontent.com/6438895/70931108-f8a35d00-2036-11ea-81a8-ea170cc5770f.png)
For me its working, I can download/upload files and the covers are displayed.
I know it's sometimes hard to convince the browsercache to display a changed cover if you upload one cover from local hard drive. But your problem sound different.

![grafik](https://user-images.githubusercontent.com/6438895/70931583-0d342500-2038-11ea-83f4-e50e95187f91.png)

**Comment by kplaczek** [comment]
I will try to upgrade RPi to buster and report back if this helped.

**Comment by kplaczek** [comment]
Ok, I have it working now. As a smart person I am of course I changed multiple things at once so I don't know which one helped. Upgraded the raspbian to buster. Removed the calibre web and installed it once again. I had it running via service and user was set to root. Changed that back to pi. Had an error that file can't be uploaded to temp directory. That was because there was an `/tmp/calibre_web` directory already and owner was set to root. I removed it and it is all working fine now.
